components/Buttons/tests/unit/MDCButtonTests.m

```diff
@@ -1340,9 +1340,10 @@ - (void)testTraitCollectionDidChangeBlockCalledWhenTraitCollectionChanges {
   MDCButton *button = [[MDCButton alloc] init];
   XCTestExpectation *expectation =
       [self expectationWithDescription:@"Called traitCollectionDidChange"];
-  button.traitCollectionDidChangeBlock = ^(UITraitCollection *_Nullable previousTraitCollection) {
-    [expectation fulfill];
-  };
+  button.traitCollectionDidChangeBlock =
+      ^(MDCButton *_Nonnull passedButton, UITraitCollection *_Nullable previousTraitCollection) {
+        [expectation fulfill];
```

Comment:
Sorry. I should have make a code snippet here like https://github.com/material-components/material-components-ios/blob/develop/components/Dialogs/tests/unit/MDCDialogPresentationControllerTests.m#L36

```
__block MDCButton *passButton;
...{
   passedButton = b;
   [expectation fulfill];
}
XCTAssertEqual(passedButton, button);
```